homeassistant/components/evohome/climate.py

```diff
@@ -334,33 +311,44 @@ def preset_mode(self) -> Optional[str]:
         """Return the current preset mode, e.g., home, away, temp."""
         return TCS_PRESET_TO_HA.get(self._evo_tcs.systemModeStatus["mode"])
 
-    async def async_set_temperature(self, **kwargs) -> None:
-        """Do nothing.
+    @property
+    def min_temp(self) -> float:
+        """Return the minimum target temperature of a evohome Controller."""
+        return None
```

Comment:
Everything has a minimum and maximum. Are there none or is it unknown ?

Reply:
I don't know if the frontend can handle this.

Reply:
It is truly `None` - the docstring is the issue.

Reply:
> I don't know if the frontend can handle this.

It seems to.